src/reducers/redemption.js

```diff
@@ -7,7 +7,7 @@ import {
     REDEMPTION_PROVE_BTC_TX_BEGIN,
     REDEMPTION_PROVE_BTC_TX_SUCCESS,
     REDEMPTION_PROVE_BTC_TX_ERROR,
-    REDEMPTION_REQUESTED
+    REDEMPTION_REQUEST_SUCCESS
```

Comment:
Fwiw, I prefer the `REDEMPTION_REQUESTED` style---but I appreciate the unification of styles between our actions, since this is currently the minority style.

Reply:
Ok, good to know for the future!